jbpm-workitems/jbpm-workitems-core/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-workitems</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>jbpm-workitems-core</artifactId>
+  <name>jBPM :: WorkItems :: Core</name>
```

Comment:
since all other jbpm/workitems-* modules depends on the -core- one during build we run into transitive depends issues. So depends for core have to be in parent pom. If there is some workaround for that please let me know. Thanks.